modules/mod_search/support/search_query.erl

```diff
@@ -336,22 +336,6 @@ parse_query([{is_published, Boolean}|Rest], Context, Result) ->
     end;
 
 
-%% is_public or is_public={false,true,all}
-%% Filter on whether an item is publicly visible or not.
-parse_query([{is_public, Boolean}|Rest], Context, Result) ->
```

Comment:
But then the ACL module should interpret this... leaning towards removing it. You can simulate it by performing the query with an unauthorized context argument.